src/hook/pauseResume.js

```diff
@@ -0,0 +1,350 @@
+			}
+
+			return view.getComputedStyle(elem);
+		};
+	} else if (document.documentElement.currentStyle) {
+		getStyles = function(elem) {
+			return elem.currentStyle;
+		};
+	}
+
+	function now() {
+		return new Date().getTime();
+	}
+
+	function AniProperty(el, prop, optall, prevProp) {
+		this.opt = optall;
+		this.start = -1;
+		this.elapsed = 0;
+		this.paused = false;
+		this.uuid = uuid++;
+		this.easingNames = [];
+		this.prop = $.extend({}, prevProp);
+		var computed;//It's only used when relative value is applied.
+
+		for (var propName in prop) {
+			var propValue = prop[propName];
+			var markIndex;
+			var sign;
+
+			//If it has a absoulte value.
+			if (typeof propValue !== "string" ||
+				(markIndex = propValue.search(/[+|-]=/)) < 0) {
+				this.prop[propName] = propValue;
+				continue;
+			}
+
+			//If it has a relative value
+			sign = propValue.charAt(markIndex) === "-" ? -1 : 1;
+
+			if (!prevProp[propName]) {
+				// If this makes performance issues, then change the way of calucating relative value.
+				computed = computed || getStyles(el);
+				prevProp[propName] = computed[propName];
+			}
+
+			this.prop[propName] = propValue
+				.replace(/([-|+])*([\d|\.])+/g,
+					generateAbsoluteValMaker(prevProp, propName, sign))
+				.replace(/[-|+]+=/g, "");
+		}
+	}
+
+	/**
+	 * Generate a new absolute value maker.
+	 *
+	 * function to avoid JS Hint error "Don't make functions within a loop"
+	 */
+	function generateAbsoluteValMaker(prevProp, propName, sign) {
+		return function absoluteValMaker(match) {
+			var prevValue = prevProp[propName];
+
+			if (!prevValue || prevValue === "auto") {
+				// Empty strings, null, undefined and "auto" are converted to 0.
+				// This solution is somewhat extracted from jQuery Tween.propHooks._default.get
+				// TODO: Should we consider adopting a Tween.propHooks?
+				prevValue = 0;
+			} else {
+				prevValue = parseFloat(prevValue);
+			}
+
+			return prevValue + (match * sign);
+		};
+	}
+
+	AniProperty.prototype.init = function() {
+		this.start = now();
+		this.elapsed = 0;
+	};
+
+	AniProperty.prototype.addEasingFn = function(easingName) {
+		this.easingNames.push(easingName);
+	};
+
+	AniProperty.prototype.clearEasingFn = function() {
+		var easing;
+		while (easing = this.easingNames.shift()) {
+			delete $.easing[easing];
+		}
+		this.easingNames = [];
+	};
+
+	function addAniProperty(el, prop, optall) {
+		var prevProp;
+		var propCount = el.__aniProps ? el.__aniProps.length : 0;
+		var newProp;
+
+		prevProp = propCount === 0 ? {} : el.__aniProps[propCount - 1].prop;
+
+		//prevProp is used for calculating absolute value by accumulating aniProps.
+		//So newProp has absolute value accumlated.
+		newProp = new AniProperty(el, prop, optall, prevProp);
+		el.__aniProps = el.__aniProps || [];
+
+		//Animation is excuted immediately.
+		if (el.__aniProps.length === 0) {
+			newProp.init();
+		}
+		el.__aniProps.push(newProp);
+	}
+
+	function getOptAll(speed, easing, fn) {
```

Comment:
@sculove Unfortunately, I've tried, but It cannot be applied. because $.speed does dequeue the fx-queue. 

Its purpose is only to make option without changing other properties(eg. fx queue). I can't find the solution not to dequeue the fx using $.speed.